packages/smooth_app/lib/pages/scan/scan_page.dart

```diff
@@ -60,86 +59,82 @@ class _ScanPageState extends State<ScanPage> {
     final bool hasACamera = CameraHelper.hasACamera;
 
     return SmoothScaffold(
-      brightness:
-          Theme.of(context).brightness == Brightness.light && Platform.isIOS
-              ? Brightness.dark
-              : null,
+      brightness: Brightness.light,
       backgroundColor:
           context.lightTheme() ? themeExtension.primaryLight : null,
-      body: SafeArea(
-        child: Column(
-          children: <Widget>[
-            if (hasACamera)
-              Expanded(
-                flex: 100 - _carouselHeightPct,
-                child: Consumer<PermissionListener>(
-                  builder: (
-                    BuildContext context,
-                    PermissionListener listener,
-                    _,
-                  ) {
-                    switch (listener.value.status) {
-                      case DevicePermissionStatus.checking:
-                        return EMPTY_WIDGET;
-                      case DevicePermissionStatus.granted:
-                        // TODO(m123): change
-                        return const CameraScannerPage();
-                      default:
-                        return const _PermissionDeniedCard();
-                    }
-                  },
-                ),
-              ),
+      body: Column(
+        children: <Widget>[
+          if (hasACamera)
             Expanded(
-              flex: _carouselHeightPct,
-              child: Padding(
-                padding:
-                    const EdgeInsetsDirectional.only(bottom: BALANCED_SPACE),
-                child: ScanPageCarousel(
-                  onPageChangedTo: (int page, String? barcode) async {
-                    if (barcode == null) {
-                      // We only notify for new products
-                      return;
-                    }
+              flex: 100 - _carouselHeightPct,
+              child: Consumer<PermissionListener>(
+                builder: (
+                  BuildContext context,
+                  PermissionListener listener,
+                  _,
```

Comment:
is that normal ? @g123k

Reply:
Yes, it was already there (marked as changed, but it's just the indentation).
`_` is when you don't want to use a parameter